scipy/stats/tests/test_stats.py

```diff
@@ -1957,12 +1967,40 @@ def test_zmap_ddof(self):
         assert_array_almost_equal(z[0], z0_expected)
         assert_array_almost_equal(z[1], z1_expected)
 
+    def test_zmap_nan_policy_omit(self):
```

Comment:
These compare the behavior of `zmap` when there are `nan`s to be ignored when the `nans` are already removed. Seems like a good test of the `omit` nan policy.